diff -uprN binutils-2.22.org/binutils/bucomm.h binutils-2.22/binutils/bucomm.h
--- binutils-2.22.org/binutils/bucomm.h	2017-03-09 12:16:31.628737239 +0100
+++ binutils-2.22/binutils/bucomm.h	2017-03-09 12:13:03.385046407 +0100
@@ -67,7 +67,7 @@ void mode_string (unsigned long, char *)
 extern void print_version (const char *);
 
 /* rename.c */
-extern void set_times (const char *, const struct stat *);
+//extern void set_times (const char *, const struct stat *);
 
 extern int smart_rename (const char *, const char *, int);
 
